content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst

```diff
@@ -80,57 +80,46 @@ quote as a new purchase order.
    the warehouse from the drop-down selection that includes the `Receipts` label at the end of the
    name.
 
-After confirming the :abbr:`PO (Purchase Order)`, a :guilabel:`Receipt` smart button will appear in
-the top of the :abbr:`PO (Purchase Order)` form — click it to reveal the associated receipt.
+After confirming the :abbr:`PO (Purchase Order)`, a :guilabel:`Receipt` smart button appears at the
+top of the :abbr:`PO (Purchase Order)` form. Click it to reveal the new receipt (WH/IN) operation.
 
-.. image:: receipts_delivery_two_steps/two-step-po-receipt.png
+.. image:: receipts_delivery_two_steps/receipts-delivery-two-steps-smart-button.png
    :align: center
    :alt: After confirming a purchase order, a Receipt smart button will appear.
 
-.. tip::
-   Purchase order receipts can also be found in the :menuselection:`Inventory` application. In the
-   :guilabel:`Overview` dashboard, click the :guilabel:`# to Process` smart button in the
-   :guilabel:`Receipts` kanban card.
-
-   .. image:: receipts_delivery_two_steps/two-step-receipts-kanban.png
-      :align: center
-      :alt: One receipt ready to process in the Inventory Overview kanban view.
-
 Process the receipt
 -------------------
 
-The receipt and internal transfer will be created once the purchase order is confirmed. The status
-of the receipt will be :guilabel:`Ready`, since the receipt must be processed first. The status of
-the internal transfer will be :guilabel:`Waiting Another Operation`, since the transfer cannot
-happen until the receipt is completed. The status of the internal transfer will only change to
-:guilabel:`Ready` once the receipt has been marked as :guilabel:`Done`.
+Once the purchase order is confirmed, a receipt is generated and ready to process.
+
+Navigate to the :menuselection:`Inventory app`, and locate the :guilabel:`Receipts` task card on the
+:guilabel:`Inventory Overview` dashboard. This reveals a list of all receipts to process.
 
 Click on the :guilabel:`Receipt` associated with the purchase order, then click :guilabel:`Validate`
-to complete the receipt and move the product to the :guilabel:`Input Location`.
+to complete the receipt and move the product to :guilabel:`WH/Input`.
 
-.. image:: receipts_delivery_two_steps/validate-two-step-receipt.png
+.. image:: receipts_delivery_two_steps/receipts-delivery-two-steps-receipt-form.png
```

Comment:
Too zoomed in, would include the larger form/UI

Reply:
Replaced with new, zoomed-out screenshot that showcases navigational UI and shows key areas. Suggestion implemented ✅